src/simplejavatexteditor/UI.java

```diff
@@ -88,7 +96,7 @@
     public UI() {
 
         // Set the initial size of the window
-        setSize(700, 500);
+        setSize(800, 500);
```

Comment:
Is there a reason why you change the size..? Didn't it fit with the changes..?

Reply:
The new buttons make the font size area dissapear, so i increased the window's size